trunk/src/modules/upscale.F90

```diff
@@ -18,83 +20,131 @@ module upscale
       integer :: NR_VERT
 !
 !  ...list of vertices' coordinates
-      real(8), allocatable :: VERTC(:,:)
+      real(8), pointer :: VERTC(:,:)
 !
 !  ...number of elements required by visualization level (0 - 3)
       integer :: NR_ELEM
 !
 !  ...list of elements' vertices (indexing starts at 0)
-      integer, allocatable :: ELEM(:,:)
+      integer, pointer :: ELEM(:,:)
+!
    endtype vis
 !
    type(vis) :: TETR_VIS, PRIS_VIS, HEXA_VIS
```

Comment:
nullify pointers on creation of these types

Reply:
Adding the nullification to `clear_vis` will do this since we always call that first.